dataline-workers/src/main/java/io/dataline/workers/JobStatus.java

```diff
@@ -25,10 +25,10 @@
 package io.dataline.workers;
 
 /**
- * Indicates whether the worker's underlying process was succesful. E.g this should return
+ * Indicates whether the worker's underlying process was successful. E.g this should return
  * SUCCESSFUL if a connection check succeeds, FAILED otherwise.
  */
 public enum JobStatus {
   FAILED,
-  SUCCESSFUL;
+  SUCCESSFUL
```

Comment:
outside the scope of this pr, but we should just call this SUCCESS. i think that will bring it in line with naming elsewhere.